### PR TITLE
standalone-community social auth

### PR DESCRIPTION
**Pull request by jctanner**
#### What is this PR doing:

Adds github social auth.

Issue: AAH-2526

### CODE REVIEW

File: galaxy_ng/app/settings.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 INSTALLED_APPS = [
     'rest_framework.authtoken',
+    'social_django',
```
Comment (rochacbruno) [inline galaxy_ng/app/settings.py:24]: Maybe this must be added on dev/standalone-community env only?
adding here will require this app to be available for all intallations

Comment (jctanner) [reply galaxy_ng/app/settings.py:24]: I agree, I'm just now sure how to do that. How did you inject that with the ldap mode? I notice it also uses the same library, which is why I picked it.

Comment (jctanner) [reply galaxy_ng/app/settings.py:24]: Oh, I found it ... https://github.com/ansible/galaxy_ng/blob/facddc251698d1ac9cbd0dbb926956b6dbb428ca/galaxy_ng/app/dynaconf_hooks.py#L122-L123

I'll get that patched in ASAP.

File: dev/standalone-community/docker-compose-ui.yml [hunk 2]
```diff
@@ -5,4 +5,4 @@ services:
     environment:
       - "API_PROXY_HOST=api"
       - "API_PROXY_PORT=8000"
-      - "DEPLOYMENT_MODE=${COMPOSE_PROFILE}"
```
Comment (rochacbruno) [inline dev/standalone-community/docker-compose-ui.yml:8]: shouldn't this be `DEPLOYMENT_MODE=standalone-community` ?

Comment (rochacbruno) [reply dev/standalone-community/docker-compose-ui.yml:8]: Actually, we have only 2 deployment modes AFAIK: standalone and insights, I think we don't want to add a new one, is that useful? or can we stick with `standalone` here?

Comment (jctanner) [reply dev/standalone-community/docker-compose-ui.yml:8]: Maybe, but all the the other UI files are using a variable:

```
[jtanner@p1 galaxy_ng]$ find dev -type f -name "*ui.yaml" | xargs fgrep -H DEPLOYMENT_MODE=
dev/insights/docker-compose-ui.yaml:      - "DEPLOYMENT_MODE=${COMPOSE_PROFILE}"
dev/standalone-keycloak/docker-compose-ui.yaml:      - "DEPLOYMENT_MODE=${COMPOSE_PROFILE}"
dev/standalone-ldap/docker-compose-ui.yaml:      - "DEPLOYMENT_MODE=${COMPOSE_PROFILE}"
dev/standalone/docker-compose-ui.yaml:      - "DEPLOYMENT_MODE=${COMPOSE_PROFILE}"
dev/standalone-community/docker-compose-ui.yaml:      - "DEPLOYMENT_MODE=${COMPOSE_PROFILE}"
```

Comment (jctanner) [reply dev/standalone-community/docker-compose-ui.yml:8]: I hardcoded it to standalone. We may need to revise later when v1 support is added depending on the capability of feature flags.

**Pull request merged**